engine/core/src/main/java/org/datacleaner/configuration/RemoteServerConfigurationImpl.java

```diff
@@ -56,5 +87,103 @@ public RemoteServerData getServerConfig(String serverName) {
         return null;
     }
 
+    @Override
+    public RemoteServerState getActualState(String remoteServerName) {
+        if (serverStatusTask == null) {
```

Comment:
There is two ways, how use it this class.
Active getting status and add Listener for future notification.
This two actions have create/check task.
If you will call only actual status again again. The task must be created.